packages/patternfly-4/react-core/tslint.json

```diff
@@ -6,6 +6,7 @@
     "tslint-config-prettier"
   ],
   "rules": {
+    "member-access": false,
```

Comment:
Setting this to true (default in strict mode) forces us to use those donky exclamation points to assert that something is not undefined at runtime (even if wrapped in an if statement or otherwise avoided...) Since we are migrating from JS to TS it makes sense to enable this as a temporary measure. See https://medium.com/@martin_hotell/react-typescript-and-defaultprops-dilemma-ca7f81c661c7